src/models/github.js

```diff
@@ -11,6 +11,24 @@ class GitHub extends Fetcher {
     return this.get('notifications')
   }
 
+  // https://developer.github.com/v3/search/#search-issues
+  getTasks(query=Config.searchQuery) {
```

Comment:
Blowing my mind with this modern JavaScript! I didn't know about default argument values.